script_library/deployment-actions-common.sh

```diff
@@ -34,6 +35,7 @@ function delete_ses_rook(){
 
 function deploy_airship(){
     echo "Deploying SUSE version of Airship"
+    setup_ssh_agent
```

Comment:
I think the ssh-agent stuff should go in a different commit or even PR as this one is dealing with the skuba/terraform packages change.

Reply:
It was subsequent error encountered after skuba/ terraform package issue in caasp v4 ECP setup. I did update commit message to reflect that. This error happens when runner host is in ECP where by default ssh-agent is not there.

But as we need to move things, I will remove this logic from here in this PR update